server.js

```diff
@@ -585,6 +585,49 @@ const ownerSocketCallbacks = {
 				Connections[user].socket.emit("sessionOptions", { boosterContent: boosterContent });
 		}
 	},
+	"setUsePredeterminedBoosters": function(userID, sessionID, value, ack) {
+		Sessions[sessionID].usePredeterminedBoosters = value;
+		Sessions[sessionID].forNonOwners(uid => Connections[uid].socket.emit("sessionOptions", { usePredeterminedBoosters: value }));
+		if(ack) ack({code: 0});
+	},
+	"setBoosters": function(userID, sessionID, text, ack) {
+		try {
+			let boosters = [];
+			let booster = [];
+			for(let line of text.split('\n')) {
+				if(!line || line === "") {
+					if(booster.length === 0) continue;
+					boosters.push(booster);
+					booster = [];
+				} else {
+					let [count, cardID] = parseLine(line);
+					if (typeof cardID !== "undefined") {
+						for (let i = 0; i < count; ++i) 
+							booster.push(getUnique(cardID));
+					} else {
+						ack(count);
+						return;
+					}
+				}
+			}
+			if(booster.length > 0) boosters.push(booster);
+
```

Comment:
Might be good for UI to check for booster errors here as well?

Reply:
I decided to wait for the startDraft event to check that since we can't know for certain how many players will participate at this point, and thus how many boosters are required.
I could check for inconsistencies in card counts though.